setup.cfg

```diff
@@ -70,7 +70,8 @@ addopts = --doctest-rst
 
 [flake8]
 max-line-length = 100
-select = E101,W191,W291,W292,W293,W391,E111,E112,E113,E502,E722,E901,E902
+# E402: Module level import not at top of file
```

Comment:
Should I selected 2 lines for the comment but I didn't. I moved this to my other code suggestion.
```suggestion
```

Reply:
Need to delete this now as suggested above. It is duplicated on L77.